rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/db/sql/DBSQLFile.kt

```diff
@@ -4,12 +4,13 @@ import com.smeup.rpgparser.interpreter.DBFile
 import com.smeup.rpgparser.interpreter.Field
 import com.smeup.rpgparser.interpreter.Record
 import com.smeup.rpgparser.interpreter.Value
-import java.lang.RuntimeException
 import java.sql.Connection
 import java.sql.ResultSet
 
 class DBSQLFile(private val name: String, private val connection: Connection) : DBFile {
```

Comment:
Given I am not familiar with this part of the code I looked into DBFile. Depending on the background of other contributors what DBFile does could be not obvious and we may want to add a few comments